tests/echo/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -267,7 +267,7 @@ echo 1000000000,
 echo 1000000000 . 'String',
     $object->property + $object->method(),
     method((method('string'))),
-    $foo ? ($bar . 'foo') + 'test' : false;
+    $foo ? $bar . 'foo' . 'test' : false;
```

Comment:
Will be invalid and output php warning, also such things are not found in real code